src/ring_theory/localization.lean

```diff
@@ -1146,6 +1150,13 @@ begin
   convert is_localization.injective _ hM hxy; simp,
 end
 
+variables {S Q M}
+
+lemma coe_submodule_le_coe_submodule (h : M ≤ non_zero_divisors R)
+  {I J : ideal R} :
+  coe_submodule S I ≤ coe_submodule S J ↔ I ≤ J :=
+submodule.map_le_map_iff_of_injective (is_localization.injective _ h) _ _
```

Comment:
Can you prove something like
```suggestion
  strict_mono (coe_submodule S)  :=
```
instead?

Reply:
I added them as separate definitions instead of replacing the `le_iff_le` version, since I plan on using those.